ci/autoresearch/debug_find_pins.py

```
@@ -0,0 +1,81 @@
+                    ):
+                        seen_response = True
+                        print(
+                            f"\n[direct] === RESPONSE in {elapsed:.3f}s ===",
+                            flush=True,
+                        )
+                        deadline = time.time() + 0.3  # drain briefly
+            time.sleep(0.005)
+
+        if not seen_response:
+            print(
+                f"\n[direct] NO RESPONSE in 10s -- firmware-side hang",
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)